app/search_builders/sufia_search_builder.rb

```diff
@@ -0,0 +1,9 @@
+class SufiaSearchBuilder < Blacklight::Solr::SearchBuilder
```

Comment:
@mjgiarlo Yes, we do not care about the permissions the current user has because they are admin, so they get to stats about all files, not just the ones they have access to. The other search builder includes permission checks.  I'll add some comments to the top